backend/kesaseteli/applications/enums.py

```diff
@@ -57,17 +57,45 @@ class YouthApplicationStatus(models.TextChoices):
     @staticmethod
     def active_values():
         """
-        Youth application status values for youth application that have been activated.
+        Youth application statuses for youth application that have been activated.
+        """
+        return [
+            YouthApplicationStatus.AWAITING_MANUAL_PROCESSING.value,
+            YouthApplicationStatus.ADDITIONAL_INFORMATION_REQUESTED.value,
+            YouthApplicationStatus.ADDITIONAL_INFORMATION_PROVIDED.value,
+            YouthApplicationStatus.ACCEPTED.value,
+            YouthApplicationStatus.REJECTED.value,
+        ]
 
-        :return: YouthApplicationStatus.values without
-                 YouthApplicationStatus.SUBMITTED.value
+    @staticmethod
+    def acceptable_values():
+        """
+        Youth application statuses from which the youth application can be accepted.
         """
         return [
-            value
-            for value in YouthApplicationStatus.values
-            if value != YouthApplicationStatus.SUBMITTED.value
+            YouthApplicationStatus.AWAITING_MANUAL_PROCESSING.value,
+            YouthApplicationStatus.ADDITIONAL_INFORMATION_REQUESTED.value,
```

Comment:
I have though that `ADDITIONAL_INFORMATION_REQUESTED` status would mean that we wait that the user will fill the "Lisätietopyyntö" form so handler should not be able to accept the application with this status. This is not very important anyway because it's not yet implemented

Reply:
Here is frontend logic for the statuses: https://github.com/City-of-Helsinki/yjdh/blob/develop/frontend/kesaseteli/shared/src/constants/status-constants.ts#L16

Reply:
> I have though that `ADDITIONAL_INFORMATION_REQUESTED` status would mean that we wait that the user will fill the "Lisätietopyyntö" form so handler should not be able to accept the application with this status. This is not very important anyway because it's not yet implemented

This is just preliminary state handling and as you said it is not yet implemented so we can define the exact allowed states later. For now this shouldn't be a problem as the system shouldn't currently create any youth applications that are in states ADDITIONAL_INFORMATION_REQUESTED or ADDITIONAL_INFORMATION_PROVIDED.

Reply:
Yes we can ignore this for now and unify the logic later